bpf/aya-bpf-macros/src/lib.rs

```diff
@@ -248,3 +248,31 @@ pub fn btf_tracepoint(attrs: TokenStream, item: TokenStream) -> TokenStream {
         .unwrap_or_else(|err| err.to_compile_error())
         .into()
 }
+
+/// Marks a function as a eBPF Socket Filter program that can be attached at
+/// a raw AF_PACKET socket.
```

Comment:
Socket filters can be attached to other sockets too not just AF_PACKET

Reply:
fixed it to say "attached to a raw socket"